.github/workflows/cicd-2-publish.yaml

```diff
@@ -58,27 +59,20 @@ jobs:
         run: |
           echo "Getting the artefacts created by the build stage ..."
           # TODO: Use either action/cache or action/upload-artifact
-      - name: "Create release"
-        id: create_release
-        uses: actions/create-release@v1
```

Comment:
Does this remove the github release itself?

Reply:
The GitHub release would be performed by the `cycjimmy/semantic-release-action@v4.1.0` action. It's configured by including the `@semantic-release/github` plugin in the `.releaserc` file.